editor/src/components/canvas/canvas-strategies/strategies/drag-to-insert-metastrategy.tsx

```diff
@@ -280,6 +281,7 @@ function dragToInsertStrategyFactory(
               reparentCommand,
               ...optionalWrappingCommand,
             ],
+            'rerender-all-elements',
```

Comment:
looks like another place to fix

Reply:
Added comment about follow up in 38ec355d5.